webrtc.html

```diff
@@ -8852,6 +8854,34 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+	  <li>If the selector is a MediaStreamTrack, return the following objects:
+	    <ul>
+	      <li>
+		The RTCMediaStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is outbound, the RTCOutboundRTPStreamStats corresponding to the selector.
+	      </li>
+	      <li>
+		At the implementation's discretion, other stats that the
+		implementation considers relevant.
```

Comment:
@alvestrand The sentence should be narrowed to *new* stats then, because as it reads now it broadly allows browsers to differ in which stats they leave out among existing agreed-upon spec'ed stats.

That said, isn't it generally understood that new specs will iterate on old ones, unless it's somehow explicitly denied?